critiquebrainz/frontend/views/artist.py

```diff
@@ -25,29 +25,42 @@ def entity(mbid):
     # Note that some artists might not have a list of members because they are not a band
     band_members = _get_band_members(artist)
 
+    reviews_limit = 5
```

Comment:
Let's make it configurable as a constant at the beginning of the file (and `release_group_limit` as well) :)

Reply:
Can I get an example of what this would look like? I've attempted placing the constants at the beginning of the file, under the imports, but I keep getting `UnboundLocalError: local variable referenced before assignment`.

Reply:
You can place then in `__init__.py` and then import the variables?

Reply:
Updated.